### PR TITLE
Document basic SEV support

### PR DESCRIPTION
**Pull request by vasiliy-ul**
Added documentation on how to run a VM with SEV memory encryption.

### CODE REVIEW

File: docs/virtual_machines/confidential_computing.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        #cloud-config
+        password: fedora
+        chpasswd: { expire: False }
+    name: cloudinitdisk
+```
+
+### Current limitations
+
+- SEV-encrypted VMs cannot contain directly-accessible host devices (that is, PCI passthrough)
+- Live Migration is not supported
```
Comment (alicefr) [inline docs/virtual_machines/confidential_computing.md:66]: We should also specify that the VMs are not attested to avoid any confusion

Comment (vasiliy-ul) [reply docs/virtual_machines/confidential_computing.md:66]: Good  point. Mentioned it as a limitation.

File: docs/virtual_machines/confidential_computing.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-fedora
+  name: vmi-fedora
+spec:
+  domain:
+    launchSecurity:
+      sev: {}
```
Comment (maiqueb) [inline docs/virtual_machines/confidential_computing.md:33]: are there any interesting parameters to be set here ? Would it make sense to elaborate a bit on it ?

Comment (vasiliy-ul) [reply docs/virtual_machines/confidential_computing.md:33]: ATM there are no additional parameters. Some will be added for SEV-{ES,SNP} and when attestation is enabled.

**Review by maiqueb — APPROVED** [review]
Thank you.

**Comment by jean-edouard** [comment]
/approve
Thank you!

**Pull request merged**